vite-plugin/src/index.ts

```diff
@@ -104,15 +117,22 @@ export default function jahia(
             build: {
               lib: {
                 // Single entry point for the client, all other files must be imported in this one
-                entry: options.client?.input ?? "./src/client/index.js",
+                entry: globSync(options.client?.input?.glob ?? "**/*.jsx", {
+                  cwd: options.client?.input?.dir ?? "./src/client/",
```

Comment:
Random question: would that work too with a space in the path (ex: `src/my client/`)? 🤔

Reply:
I guess so, it's the glob library that takes care of it, spaces in names are valid